### ISSUE TITLE
[Q&A] Old 8x32 panel without a,b,c.. lines

### ISSUE THREAD

**Issue by Carlosc46** [hzeller/rpi-rgb-led-matrix#1630]
Hello, how could I make a panel with 8 rows and 32 columns work that does not have lines a,b,c...
it is composed entirely of shift registers, 4 rows and 32 lines work with r0, g0, and b,0 and the other 4 rows and 32 lines with r1, g1, b1, to fill them you would have to push 4*32 = 128 bits for each data line (treat 4 lines as 1).
I have tried various configurations and have not been able
The scheme is the following.
```
rows            subpanel 0 - r1, g1. b1
            ----------------------------------------
0          | 1   2   3   4     17  18  19  20    33  34  35   ..
1          | 5   6   7   8     21  22  23  24    37  38  ...
2          | 9   10  11  12    25  26  27  28    41 ...
3          | 13  14  15  16    29  30  31  32    until reaching 128

                    subpanel 1 - r2, g2. b2
            ----------------------------------------
4          | 1   2   3   4     17  18  19  20    33  34  35   ..
5          | 5   6   7   8     21  22  23  24    37  38  ...
6          | 9   10  11  12    25  26  27  28    41 ...
7          | 13  14  15  16    29  30  31  32    until reaching 128
```

I have tried several configurations and I have not been able to, the tests with solid colors and dimmer work, but when I use lines or text the pixels are positioned incorrectly and I also notice how only the first 32 LEDs of the two 4x4 groups fill up because they always push 32 bits.
In short, I need to receive the bits of 4 lines in 1 sentence
Thanks to everyone, they have done a great job

**Comment by Carlosc46** [comment]
I have solved it, I have modified the code, removed everything related to rowAddres from DumpToMatrix and let it write the bits through the channels that interested me, again, thank you for your work
